charts/datasaur/templates/jobs/mysql-upgrade-job.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  labels:
+    app: {{ .Release.Name }}-mariadb-upgrader
+  name: {{ .Release.Name }}-mariadb-upgrader
+spec:
+  ttlSecondsAfterFinished: 30
+  template:
+    metadata:
+      labels:
+        app: {{ .Release.Name }}-mariadb-upgrader
+      annotations:
```

Comment:
will there be a side effect because you overriding the post-install hook?

e.g. when the helm chart is uninstalled, will the job be left over?

Reply:
yes it will be over even when not uninstalled. I set the job to remove itself when finished @tonoman3g